stackit/internal/services/ske/cluster/resource.go

```diff
@@ -46,6 +47,8 @@ const (
 	VersionStateSupported        = "supported"
 	VersionStatePreview          = "preview"
 	VersionStateDeprecated       = "deprecated"
+
+	SKEUpdateDoc = "SKE automatically updates the cluster Kubernetes version if you have set `maintenance.enable_kubernetes_version_updates` to true or if there is a mandatory update, as described in `Updates for Kubernetes versions and Operating System versions in SKE`(https://docs.stackit.cloud/stackit/en/version-updates-in-ske-10125631.html)."
```

Comment:
Should this be a link, like this
```suggestion
	SKEUpdateDoc = "SKE automatically updates the cluster Kubernetes version if you have set `maintenance.enable_kubernetes_version_updates` to true or if there is a mandatory update, as described in [Updates for Kubernetes versions and Operating System versions in SKE](https://docs.stackit.cloud/stackit/en/version-updates-in-ske-10125631.html)."
```
?

Reply:
good point :+1: